docs/wip/theme-shadowing.md

```diff
@@ -1,27 +1,28 @@
-This documentation will address Shadowing in Gatsby Themes focusing on how we use it in the Store Framework Jamstack (SFJ), you can find the Shadowing concept general explanations and examples in the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/).
+This documentation will address Shadowing in Gatsby Themes focusing on how we use it in the Store Framework Jamstack (SFJ). You can find general explanations and examples of the Shadowing concept in the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/).
 
 ## Importance of Shadowing
-In [Gatsby documentation](https://www.gatsbyjs.com/docs/conceptual/plugins-themes-and-starters/) we see that _"Gatsby themes allow Gatsby site functionality to be packaged as a standalone product for others (and yourself!) to easily reuse."_, this means that when you use the theme we developed you will be able to use pre-configured functionality, data sourcing, design and everything else we add to our Store Framework Theme.
+In [Gatsby documentation](https://www.gatsbyjs.com/docs/conceptual/plugins-themes-and-starters/) we see that _"Gatsby themes allow Gatsby site functionality to be packaged as a standalone product for others (and yourself!) to easily reuse."_, it means that when you use the theme we developed, you will be able to use pre-configured functionality, data sourcing, design and everything else we add to our theme.
 
-Our theme package already includes a site structure with a handful of pages and React components that are used to build this pages. You can use this structure to guide your site specifics pages and components. In order to be able to change what we made available, you can make use of the concept of Shadowing.
+Our theme packages already includes a site structure with a handful of pages and React components that are used to build those pages. You can use this structure to guide the development of specific pages and components of your site. In order to be able to change what we offer, you can make use of the Shadowing concept.
 
-As stated in the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/): _"This feature allows users to replace a file in the src directory that is included in the webpack bundle with their own implementation."_. Therefore, you have the ability to adapt any file to suit your needs, for example: styles, to match your visual identity, and even change the functionality of the component, to meet your business rules.
+As stated in the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/): _"This feature allows users to replace a file in the src directory that is included in the webpack bundle with their own implementation."_. Therefore, you have the ability to adapt any file to suit your needs, for example: styles, to match your visual identity, and even change the functionality of a component, to meet your business rules.
```

Comment:
As stated in the [Gatsby documentation](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/):
 > This feature allows users to replace a file in the src directory that is included in the webpack bundle with their own implementation.
 
You'll have the ability to adapt any file to suit your needs, for example: styles, to match your visual identity, and even change the functionality of a component, to meet your business rules.